### DISCUSSION TITLE
customizing sidebar label for a page

### DISCUSSION THREAD

**Discussion by ryanwaits** [fuma-nama/fumadocs#307] [Q&A] [answered]
is there a way, similar to `sidebar_label` frontmatter in docusaurus, where you can control the display title for a page on the sidebar versus on the page itself?

Example:

```mdx
---
title: Let's get started
sidebar_label: Get started
description: ...
---
```

**Comment by fuma-nama** [comment, marked as answer]
You can remove, or customise the title element in the page (generally located in `/docs/[[...slugs]]/page.tsx`). You may add a new field to the page schema as well.

**Reply by ryanwaits** [reply]
ah nice, appreciate it

**Reply by sheldygg** [reply]
Hi, could you share example how i can add a new field to the page schema?
I want disable TOC for some pages.

**Reply by fuma-nama** [reply]
> Hi, could you share example how i can add a new field to the page schema?
> 
> I want disable TOC for some pages.

Depends on your source provider, for fumadocs mdx, please read the docs.

**Reply by sheldygg** [reply]
I read all sections from this page

https://fumadocs.vercel.app/docs/mdx

Don't see how i can add new fields.
So could you share link?

**Reply by fuma-nama** [reply]
Actually you can open the sidebar and see other items.
https://fumadocs.vercel.app/docs/mdx/validation

**Reply by sheldygg** [reply]
Still don't understand how i can disable TOC for some pages.
